src/carousel.css.ts

```diff
@@ -88,6 +88,7 @@ const arrowButtonBase = style([
 ]);
 
 export const carouselContainer = sprinkles({
+    isolation: 'isolate',
```

Comment:
this fixes a zIndex issue with carousel arrows and Sheet
https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AlGBnADhAduglgG4wAyAhgJ4QCuALgHwA6OABC0gMJkBON6MAG2Zs2Ad3y0AFgCFqAgTFrphIiTAC26ALzAAgt26UAdADNe6gBTAWCnAHMp8FgDYWAXwA0LCwH0v%2BKAAPAEoWLXpvFRF2AFkYKHwyLm4oKOiWSRgyKAF8HBgdJAAVMjsWWgpMAsYQTHMIGvoACSycvJgkAHoSu3o3NOjaCQUdGuSoFhqWAGoWAMD%2B1nS2WHQwbnxMIdwtGoARDHXN7ZwagZF1eMTRpeW2JABJdVKYc%2BX0bjBdkElaWkx0PBOp10DRPjAjNQ8JgBGR0JIjJB1J0AIzOAAM6MCAE5MZ0APyiMi0MCSM63O5sOFVMC0FDE-AQb5o%2BDY8mUtidJgU6KLSkAIzotB2wDeIiQsj%2BuAAChtntwKCx0M95CwZdwMNpgBZQuEWMA3G5uRyRLpaYzTjz0l1JcKcLL8PKKMa7ny7oKpTgSHkANaFW24b04H1q%2B0a9BanVhCIGo1Bn02oWB319c5ctLBYJu9M4LpoLC4AjEchUOjc5gSpNhx08CgqBLoMj8hRQHR2RQAZVoxJgFhq%2BHQHcyigA8lVTiAsyp1ZqdFG9aKKfxaF2e32QAOhzBR%2BOal5aNxqDBggBuFSG5guscwS2Jz0Op3lnDAdsr7u0Xv9wfD2jXiehAAyADIiWJAt0UNJcA4AQIH4OddRjc5l1XD9103H8-z3FgTDIAR%2BFPNILyWF17nzbA8CIUhKBoBhzglCBAjFTBsgSexpAgKV1B0ax1Agfl8AUJw0S8VYfWFTAnAAFnRdw3WWZioFYuwiggTBuJYXj%2BMElh0REjAxNUqSZKI5YSOtaVYTAGBJAgARYG4Fgc2WG0GLMvMMHIosqNLWjQM6cDfKnHAQA8EApA0DAEAAbRAGIIEIAdu24EAAF1QvEKApHQaKAGYMQ8FF0QAJkklK3CAA